www/src/pages/pt/usage/tailwind.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Tailwind CSS
+description: Uso do Tailwind CSS
+layout: ../../../layouts/docs.astro
+lang: pt
+---
+
+## O que é Tailwind CSS?
+
+Tailwind CSS é um pequeno framework CSS focando em [classes utilitárias](https://tailwindcss.com/docs/utility-first) para criar designs personalizados, sem a troca de contexto que o CSS normal exige. É puramente uma estrutura CSS e não fornece nenhum componente ou lógica pré-criada, e fornece [um conjunto muito diferente de benefícios](https://www.youtube.com/watch?v=CQuTF-bkOgc) em comparação com um biblioteca de componentes como Material UI.
+
+Isso torna o CSS incrivelmente fácil e rápido de escrever, conforme mostrado no exemplo a seguir:
+
+CSS Antigo:
+
+1. Escreva CSS, geralmente em um arquivo separado
+
+```css
+.minha-classe {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+  border: 1px solid #e2e8f0;
+  border-radius: 0.25rem;
+  padding: 1rem;
+}
+```
+
+2. Import CSS into your component
+
+```jsx
+import "./minha-classe.css";
+```
+
+3. Add the class to your HTML
```

Comment:
This phrase is in english

Reply:
solved in commit [a308d54](https://github.com/t3-oss/create-t3-app/pull/853/commits/a308d54a5e8f7957194b870602fa8cf2728f7e7a)